README.md

```
@@ -284,177 +362,187 @@ Namespaces and tables:
+  - `group` (Address)
+  - `to` (Address)
+  - `id` (BigInteger)
+  - `value` (BigInteger)
+- `GroupRedeemCollateralBurn`
+  - `group` (Address)
+  - `id` (BigInteger)
+  - `value` (BigInteger)
 
 #### V_CrcV1 (Circles v1 views)
+
     * `Avatars` (view combining `Signup` and `OrganizationSignup`)
```

markdownlint-cli2 (0.18.1)

Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)